app/controllers/resources_controller.rb

```diff
@@ -205,6 +229,16 @@ def signed_id?(file_id)
     ActiveStorage.verifier.valid_message?(file_id)
   end
 
+  def globus_id?(file_id)
+    file_id =~ %r{globus://}
+  end
+
+  def decoratable_id?(file_id)
+    return false unless signed_id?(file_id) || globus_id?(file_id)
+
+    true
```

Comment:
🤔 is this functionally equivalent to

```
signed_id?(file_id) || globus_id?(file_id)
```

? If so, I think we can remove the explicit booleans and conditional.